rkt/image/io.go

```diff
@@ -114,6 +115,19 @@ func getTmpROC(s *store.Store, path string) (*removeOnClose, error) {
 	if err != nil {
 		return nil, errwrap.Wrap(errors.New("error setting up temporary file"), err)
 	}
+	// let's lock the file to avoid concurrent writes to the temporary file, it
+	// will go away when removing the temp file
+	_, err = lock.TryExclusiveLock(tmp.Name(), lock.RegFile)
+	if err != nil {
+		if err != lock.ErrLocked {
+			return nil, err
```

Comment:
Maybe add some text describing what failed? "failed to lock a file" or something?